tests/test_preprints.py

```diff
@@ -462,13 +467,114 @@ def test_format_preprint(self):
 
         assert nodes == {}
 
+    def test_format_thesis(self):
```

Comment:
After speaking to you and reviewing the test, I am simply adding a description of the test at the top of the test. While there is an overwhelming amount of assertions and list comprehension going on, the big picture is pretty simple.

```format_preprint``` takes a ```PreprintService``` and creates a ```GraphNode``` appropriate for uploading to SHARE. This test, compares the ```PreprintService``` Object and the resulting ```GraphNode``` for accuracy.